src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    protected String color;
```

Comment:
```suggestion
    public String color;
```

Reply:
If I make it 'private', I need to create getter, so I made it 'protected'. If it nesecerally must be 'private' I'll fix it.